diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -428,8 +430,13 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule(
     private fun handleValueParameterList(node: ASTNode, entryType: String) = node
         .children()
         .filter {
-            it.elementType == COMMA &&
-                    !it.treeNext.run { elementType == WHITE_SPACE && textContains('\n') }
+            (it.elementType == COMMA && !it.treeNext.isNewLineNode()) ||
+                    // Move RPAR to the new line, if there is exist return type == TYPE_REFERENCE
```

Comment:
I think, return type is not required here. IF function returns Unit, then it still should follow this style:
```kotlin
fun foo(arg1: Foo,
            arg2: Foo
) {
    // body
}
```

Reply:
I thought about this, and the first version were used such logic: append new line anytime before `RPAR`

But while I changed the code according this rule over the project, I met, that probably such cases seems appropriate without new line. 

Since in this case, we don't need to point out to the reader to something important by adding this extra line, because there are only brackets on it

For example let take less `test-based` example, than yours, the following seems to me more prettier

```kotlin
private fun deleteSpaces(node: ASTNode,
                         rightSide: Boolean) {
```

Than this:
```kotlin
private fun deleteSpaces(node: ASTNode,
                         rightSide: Boolean
) {
```

But, if you are still disagree with me, I can revert logic and add new lines anytime

Reply:
Ok, but what about this code:
```kotlin
private fun deleteSpaces(
    node: ASTNode,
    rightSide: Boolean) {
    // method body starts here
}
```
vs this:

```kotlin
private fun deleteSpaces(
    node: ASTNode,
    rightSide: Boolean
) {
    // method body starts here
}
```
I think in the latter case parameter list and method body are separated more clearly.

Reply:
Ok, don't check for return type for now, add new line after `RPAR` always